starry_night/skycam.py

```diff
@@ -106,11 +108,21 @@ class TooEarlyError(Exception):
     pass
 
 def get_last_modified(url, timeout):
```

Comment:
I think this function should raise if it was not able to get the timestamp. But Maybe the same exception.